README.md

```diff
@@ -83,6 +83,38 @@ python3 manage.py runserver
 ```
 Now open localhost:8000 in the browser
 
+### Enable HTTPS connections
+
+Certain features (example: GPS location collection) only work with HTTPS connections.  To enable HTTPS connections follow the below steps.
+
+Create self-signed certicate with openssl
+
+```
+$openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 365 -subj '/CN=localhost' -nodes
+```
+[https://stackoverflow.com/questions/10175812/how-to-create-a-self-signed-certificate-with-openssl#10176685]
+
+Install django-sslserver
+
+```
+$pip3 install django-sslserver
+```
+
+Update INSTALLED_APPS with sslserver by editing the file floodrelief/settings.py (diff below)
```

Comment:
here add `(remove this while pushing back to git)`

Reply:
addressed this change.